src/cameras/src/pm/Camera.cpp

```diff
@@ -538,6 +538,10 @@ std::string pm::Camera<F>::GetError() const {
     return message;
 }
 
+template<FrameConcept F>
+uns32 pm::Camera<F>::GetFrameBytes(){
```

Comment:
this should probably check if `ctx` is valid first, something like, and change the function so it returns a bool and takes a reference to a `uns32` value so an error can be returned
```
if (!ctx) {
        spdlog::error("Some error message");
        return false;
    }
```

Reply:
actully, `m_camera->ctx` is a public member variable so you don't need to add a function to access it from `MainWindow` you can just use `m_camera->ctx->frameBytes` instead of adding this function